app/AppKernel.php

```diff
@@ -28,7 +28,6 @@ public function registerBundles()
             new Symfony\Bundle\SecurityBundle\SecurityBundle(),
             new Symfony\Bundle\TwigBundle\TwigBundle(),
             new Symfony\Bundle\MonologBundle\MonologBundle(),
-            new Symfony\Bundle\AsseticBundle\AsseticBundle(),
```

Comment:
In addition to the removed code in this commit also please remove any other Assetic config references. I grepped and found these:

- https://github.com/OpenConext/Stepup-SelfService/blob/7a46375d0a2d83c3cac911114f07d9d3d0bd47fd/app/config/config_dev.yml#L36-L37
- https://github.com/OpenConext/Stepup-SelfService/blob/996ed6af12794b2492eeb7e3a6db327c810f3f57/composer.json#L30
- https://github.com/OpenConext/Stepup-SelfService/blob/5b7f50540b95756eabd4e5a9f6eda6855e5d2602/app/config/parameters.yml.dist#L11